MessageBusReader/Program.cs

```diff
@@ -15,6 +17,13 @@ class Program
         private static ServiceBusClient _client;
         private static ServiceBusProcessor _processor;
 
+        // Customise your operational params.
+        private const string QueueName = "error";
+        private const string Env = "DEV_CONNECTION_STRING";
```

Comment:
I would expect a variable called `Env` to have values "dev", "qa" and "prod". We are conflating the environment with the only value (at present) that changes by environment. I think we can do this better?